qa/workunits/rbd/cli_generic.sh

```diff
@@ -505,6 +505,41 @@ test_deep_copy_clone() {
     remove_images
 }
 
+test_clone_v2() {
+    echo "testing clone v2..."
+    remove_images
+
+    rbd create $RBD_CREATE_ARGS -s 1 test1
+    rbd snap create test1@1
+    rbd clone --rbd-default-clone-format=1 test1@1 test2 && exit 1 || true
+    rbd clone --rbd-default-clone-format=2 test1@1 test2
+    rbd clone --rbd-default-clone-format=2 test1@1 test3
+
+    rbd snap protect test1@1
+    rbd clone --rbd-default-clone-format=1 test1@1 test4
+
+    rbd children test1@1 | sort | tr '\n' ' ' | grep -E "test2.*test3.*test4" > /dev/null 2>&1
```

Comment:
nit: you could use 'grep -q' instead of redirect if you don't want the output (though personally I like to see the grep result in logs: it might be useful when troubleshooting a test failure).